content/en/docs/Tracing/jaeger-walkthrough.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Tracing Tutorial with Jaeger
+description: A short proof-of-concept tutorial of tracing using Jaeger.
+weight: 10
+---
+
+This walkthrough shows how to create a local cluster and deploy a number of components, including an [ingress-nginx](https://github.com/kubernetes/ingress-nginx) ingress controller, and [Jaeger](https://www.jaegertracing.io) to store, query and visualise traces.
+
+On the prepared cluster we will deploy Kyverno with tracing enabled and a couple of policies.
+
+Finally we will exercise the Kyverno webhooks by creating a Pod, then we will use Grafana to find and examine the corresponding trace.
```

Comment:
Copy-and-paste error: Not using Grafana here.

Reply:
Fixed